templates/barbican/config/barbican-api-config.json

```diff
@@ -78,14 +78,16 @@
       "optional": true,
       "merge": true
     },
```

Comment:
this will not be valid json when `P11ClientDataPath` is not set and this element ends with `,`. could just move the conditional entry somewhere above to not be the last one.

Reply:
No longer conditional